libs/blocks/global-navigation/global-navigation.js

```diff
@@ -993,6 +995,18 @@ class Gnav {
       const elements = [...document.querySelectorAll('.feds-localnav .feds-navItem')].find(
         (el) => el.textContent.trim() === navItem.textContent,
       );
+      const allLocalnavLinks = [...document.querySelectorAll('.feds-localnav button, .feds-localnav a, .feds-localnav .feds-menu-headline')];
```

Comment:
There's a specific keyboard navigation file, should that go in here or the more general global nav ?

Reply:
@mokimo Moved this piece of code as part of keyboard navigation by creating a separate file specifically for LocalNav.